Resources/AzureDevOps/VariableTemplates/DevEnvironmentVariables.yml

```diff
@@ -8,4 +8,6 @@ variables:
 - name: CosmosDatabaseThroughput
   value: 400
 - name: ReleaseEnvironmentName
-  value: DEV
\ No newline at end of file
+  value: DEV
+- name: ApiProductDisplayName
```

Comment:
I think this should be defined in a new SharedEnvironmentVariables.yml file rather than added to the existing file 3 times.  Duplicating things in our APIM config has caused us problems in the past

Reply:
Refactored shared variables into shared var set